instance/models/openedx_instance.py

```diff
@@ -213,17 +213,22 @@ def get_load_balancer_configuration(self, triggered_by_instance=False):
             )
         return backend_map, backend_conf
 
-    def set_active_vm_dns_records(self):
+    def set_active_vm_dns_record(self, appserver_num=1, ip_addr=None):
         """
-        Set DNS A records for all active app servers.
+        Set DNS A record for the active app server.
         """
-        self.logger.info("Setting DNS records for active app servers...")
-        for i, appserver in enumerate(self.get_active_appservers(), 1):
```

Comment:
@pkulkark, wouldn't it be a good and correct idea to use the existing logic and just remove the `vm<N+1>.<instance domain>` record? The removal method doesn't throw an error when trying to delete a non-existing record.

Reply:
Do you mean like just loop over all appservers for an instance and remove the `vm<i>.<instance domain>` record? Wouldn't that add unnecessary overhead? Say for example, if we archive an instance that has a large number of appservers but only a couple that were activated?